package.json

```diff
@@ -67,7 +67,7 @@
     "bip39-light": "^1.0.7",
     "ethereumjs-util": "^6.1.0",
     "ethereumjs-wallet": "^0.6.3",
-    "gun": "0.2019.915",
+    "gun": "git+https://github.com/GoodDollar/gun.git#master",
```

Comment:
this is not required.
master brach is already fixed you should not need to do anything besides making this PR off the master branch